{
  "configurations": {
    "CVE_data_version": "4.0",
    "nodes": []
  },
  "cve": {
    "CVE_data_meta": {
      "ASSIGNER": "cve@mitre.org",
      "ID": "CVE-2020-5014"
    },
    "data_format": "MITRE",
    "data_type": "CVE",
    "data_version": "4.0",
    "description": {
      "description_data": [
        {
          "lang": "en",
          "value": "IBM DataPower Gateway V10 and V2018 could allow a local attacker with administrative privileges to execute arbitrary code on the system using a server-side requesr forgery attack. IBM X-Force ID: 193247."
        }
      ]
    },
    "problemtype": {
      "problemtype_data": [
        {
          "description": []
        }
      ]
    },
    "references": {
      "reference_data": [
        {
          "name": "ibm-datapower-cve20205014-ssrf (193247)",
          "refsource": "XF",
          "tags": [],
          "url": "https://exchange.xforce.ibmcloud.com/vulnerabilities/193247"
        },
        {
          "name": "https://www.ibm.com/support/pages/node/6426789",
          "refsource": "CONFIRM",
          "tags": [],
          "url": "https://www.ibm.com/support/pages/node/6426789"
        }
      ]
    }
  },
  "impact": {},
  "lastModifiedDate": "2021-03-08T21:27Z",
  "publishedDate": "2021-03-08T18:15Z"
}